docs/literate/src/files/first_steps/changing_trixi.jl

```diff
@@ -62,6 +75,25 @@
 # if already inside the `run` directory.
 
 
+# ## Developing Trixi.jl
+
+# If you've created and cloned your own fork of Trixi.jl, you can make local changes to Trixi.jl
+# and propose them as a `Pull Request` to be merged into `trixi-framework/Trixi.jl`.
+
+# Linux and MacOS utilize the `git` version control system to manage changes between your local and
```

Comment:
Thank you for the review!

I haven't try, but I believe it's possible. However, the previous part of this tutorial, regarding the cloning of the repository, was also written with the usage of GitHub Desktop. Personally, I find GitHub Desktop more comfortable to use under Windows since Windows users may not interact with a Terminal as frequently...

Reply:
Ok. I'm fine with how it is now.